app/src/main/java/com/example/parkingsystemjava/mvp/presenter/ParkingPresenter.java

```diff
@@ -0,0 +1,21 @@
+package com.example.parkingsystemjava.mvp.presenter;
+
+import com.example.parkingsystemjava.mvp.contract.MainActivityContract;
+import com.example.parkingsystemjava.mvp.model.ParkingModel;
+import com.example.parkingsystemjava.mvp.view.ParkingView;
+
+public class ParkingPresenter implements MainActivityContract.MainActivityPresenter {
+
+    private MainActivityContract.MainActivityModel model;
+    private MainActivityContract.MainActivityView view;
+
+    public ParkingPresenter(ParkingModel model, ParkingView view) {
```

Comment:
here you should receive the contracts as parameters

Reply:
done!